packages/echarts-demo/src/bar-chart.js

```diff
@@ -98,6 +98,7 @@ const getAhoiConfig = () => {
 }
 
 const registerEventListener = () => {
+  showOnboarding = !showOnboarding;
   const helpIcon = document.getElementById("show-onboarding");
   if(!helpIcon) { return; }
   helpIcon.addEventListener('click', async () => {
```

Comment:
Moving up the `showOnboarding = !showOnboarding;` from line 110 to line 104 (before the if) should do the trick without changing the flag when the event listener is registered. Please change it for all demos. You can then remove line 101.